server/api/order/order.controller.js

```diff
@@ -22,10 +23,28 @@ exports.show = function(req, res) {
 
 // Creates a new order in the DB.
 exports.create = function(req, res) {
-  Order.create(req.body, function(err, order) {
-    if(err) { return handleError(res, err); }
-    return res.json(201, order);
-  });
+  console.log(req.body.total);
+  var charge = stripe.charges.create({
+      amount: req.body.total,
+      currency: 'usd',
+      card: req.body.billing.stripeToken,
+      description: "email@email.com"
+    }, function(err,charge) {
+          if(err && err.type === 'StripeCardError') {
```

Comment:
ah yes the StripeCardError is if the card is declined...the other is just an err in general.  I found that if the card was accepted but the amount on the charge was not a number it would still get past that first conditional.  I threw in the second one so I could catch the not a number result.  I think you would send two different messages back to the frontend so I should change that.